src/routes/root.jsx

```diff
@@ -1,31 +1,10 @@
+/* eslint-disable */
```

Comment:
The error suppressed is due to deleting an import statement of one of the components used in the file. which is causing a rendering error.

In simpler words the changes on this file is causing the application to crash.